### PR TITLE
Adding blog to site

### PR DESCRIPTION
**Pull request by eang0521**

### CODE REVIEW

File: posts/apprenticeship.md [hunk 1]
```diff
@@ -1,28 +1,32 @@
...
-
-The first project that A&R has  been focused on is called Maskil; envisioned to be the FIRST EVER digital hymnal and worship study Bible. (Imagine every worship song, every language, with (Rap)Genius-style inline annotations and two-way links to Bible passages). Currently Maskil has over 1000 songs in English, 650+ songs in Korean, and growing!.
-
-We have a feature roadmap planned out for years. You will be working collaboratively with a small team of engineers. Some of the technologies you will learn and be exposed to include: 
-- Django
-- React
-- MySQL
-- LESS CSS
-- Git Workflow
+---
```
Comment (jontsai) [inline posts/apprenticeship.md:1]: I think this should just be a regular page under `pages/apprenticeship.md` -- what do you think?

Comment (eang0521) [reply posts/apprenticeship.md:1]: This was a file that was already in the code. We just used it as a test page. We can put the posts in the pages directory if you would prefer that.

Comment (jontsai) [reply posts/apprenticeship.md:1]: Yes, please. I must have missed it during the original implementation.

However, if it breaks the Markdown rendering, we can hold off and save it for a separate PR.

Actually, now that I look at it, it's somewhat odd in that we have two files:
- https://github.com/aweandreverence/www.aweandreverence.com/blob/master/pages/apprenticeship.js
- https://github.com/aweandreverence/www.aweandreverence.com/blob/master/pages/posts/apprenticeship.md

It seems that only one of them is being used.

I think we should just delete `apprenticeship.md`

File: pages/posts/[id].js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import Page from '../../components/base_page';
+import { getAllPostIds, getPostData } from '../../components/posts_data';
+import Date from '../../components/date';
+
+import css from '../../styles/common.module.scss';
+
+import Head from 'next/head';
+
+export default function Post({ postData }) {
```
Comment (jontsai) [inline pages/posts/[id].js:9]: How will the URL appear? For SEO (https://en.wikipedia.org/wiki/Search_engine_optimization) purposes, it's important that the URL be human-readable and not merely a bunch of numbers.

Comment (eang0521) [reply pages/posts/[id].js:9]: The URL is www.aweandreverence.com/posts/test

File: posts/apprenticeship.md (hunk 1 above)
Comment (eang0521) [reply posts/apprenticeship.md:1]: Sounds good. I'll do that tonight.

**Comment by eang0521** [comment]
image

Image of the current blog landing page. We'll add styling once we confirm everything is working.

**Review by jontsai — APPROVED** [review]
LGTM

**Pull request merged**